zivid_camera/src/zivid_camera.cpp

```diff
@@ -381,19 +428,14 @@ bool ZividCamera::isConnectedServiceHandler(IsConnected::Request&, IsConnected::
 void ZividCamera::publishFrame(Zivid::Frame&& frame)
 {
   const bool publish_points = points_publisher_.getNumSubscribers() > 0 || use_latched_publisher_for_points_;
```

Comment:
Suggest extracting to a function `shouldPublishPoints()` for consistency with `shouldPublishColorImg()`

Reply:
I have already started testing all the samples, so I can deal with that.

Reply:
All samples and tests run fine here.